layouts/shortcodes/cli/influxd-flags.md

```diff
@@ -1,35 +1,39 @@
+|      | `--bolt-path`                                               | Path to boltdb database (default `~/.influxdbv2/influxd.bolt`)                                                                                                                                                                                                             | string     | `INFLUXD_BOLT_PATH`                  |
+|      | `--e2e-testing`                                             | Add /debug/flush endpoint to clear stores; used for end-to-end tests (default `false`)                                                                                                                                                                                     |            | `INFLUXD_E2E_TESTING`                |
+|      | `--engine-path`                                             | Path to persistent engine files (default `~/.influxdbv2/engine`)                                                                                                                                                                                                           | string     | `INFLUXD_ENGINE_PATH`                |
+| `-h` | `--help`                                                    | Help for the `influxd` command                                                                                                                                                                                                                                             |            |                                      |
+|      | `--http-bind-address`                                       | Bind address for the REST HTTP API (default `:8086`)                                                                                                                                                                                                                       | string     | `INFLUXD_HTTP_BIND_ADDRESS`          |
+|      | `--influxql-max-select-buckets`                             | Maximum number of group by time bucket a SELECT can create. A value of zero will max the maximum number of buckets unlimited.                                                                                                                                              | integer    |                                      |
+|      | `--influxql-max-select-point`                               | Maximum number of points a SELECT can process. A value of 0 will make the maximum point count unlimited. This will only be checked every second so queries will not be aborted immediately when hitting the limit                                                          | integer    |                                      |
```

Comment:
SELECT statement/query can process

will make the maximum point count unlimited >> allows an unlimited number of points?


@stuartcarnie re this statement:
_This will only be checked every second so queries will not be aborted immediately when hitting the limit._
Does this mean that the maximum number of points is checked every second and when the maximum number is exceeded, then a query aborts?

Reply:
Yes, that is correct, so when the system checks on a given second interval, if that number exceeds the maximum selected points, the query will abort at that time.